drv/lpc55-spi/src/lib.rs

```diff
@@ -195,7 +195,8 @@ impl Spi {
         rxignore: bool,
         len_bits: u8,
     ) {
-        if len_bits > 16 || len_bits < 4 {
+        // SPI hardware only supports lengths of range 4-16 bits
+        if !(4..=16).contains(&len_bits) {
```

Comment:
This looks correct, but I'm not sure it's an improvement in readability, particularly because of the negation. :shrug:

Reply:
This can be an assert, right and get compiled out for production.

Reply:
I don't think so – `assert!` remains in both debug and release builds; `debug_assert!` is only included in debug builds, but we never make debug builds 😆